examples/war_logs.py

```diff
@@ -1,10 +1,13 @@
-import asyncio
 import os
 
 import coc
 
 # email and password is your login credentials used at https://developer.clashofclans.com
-client = coc.login(os.environ["DEV_SITE_EMAIL"], os.environ["DEV_SITE_PASSWORD"], key_names="coc.py tests")
+client = coc.login(
```

Comment:
Same comment as above re. `.get`, also this doesn't need to be separated into 3 lines - it is less than 120chars.